iroh-net/src/discovery/dns.rs

```diff
@@ -14,10 +14,14 @@ pub const N0_DNS_NODE_ORIGIN_PROD: &str = "dns.iroh.link";
 /// The n0 testing DNS node origin, for testing.
 pub const N0_DNS_NODE_ORIGIN_STAGING: &str = "staging-dns.iroh.link";
 /// Testing DNS node origin, must run server from [`crate::test_utils::DnsPkarrServer`].
-#[cfg(any(test, feature = "test-utils"))]
-#[cfg_attr(iroh_docsrs, doc(cfg(any(test, feature = "test-utils"))))]
+#[cfg(test)]
 pub const TEST_DNS_NODE_ORIGIN: &str = "dns.iroh.test";
 
+/// Environment variable to force the use of staging relays.
+#[cfg(not(test))]
+#[cfg_attr(iroh_docsrs, doc(cfg(not(any(test, feature = "test-utils")))))]
+const ENV_FORCE_STAGING_RELAYS: &str = "IROH_FORCE_STAGING_RELAYS";
```

Comment:
You're adding this to more than one file.  Let's add it to `src/relays.rs` and make it pub so we can document this as well.